src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/IverksettMotOppdragTask.kt

```diff
@@ -20,36 +21,38 @@ import java.util.*
 class IverksettMotOppdragTask(val hentIverksettService: HentIverksettService,
                               val oppdragClient: OppdragClient,
                               val taskRepository: TaskRepository,
-                              val lagreTilstandService: LagreTilstandService
+                              val lagreTilstandService: LagreTilstandService,
+                              val hentTilstandService: HentTilstandService
 ) : AsyncTaskStep {
 
     override fun doTask(task: Task) {
         val behandlingId = UUID.fromString(task.payload)
         val iverksett = hentIverksettService.hentIverksett(behandlingId)
-        val forrigeTilkjentYtelse = null // TODO: Hent ut forrigeTilkjentYtelse fra lokal DB
+        val forrigeTilkjentYtelse = iverksett.behandling.forrigeBehandlingId?.let { hentTilstandService.hentTilkjentYtelse(it) }
```

Comment:
bør vi legge til en `?: error("Kunne ikke finne tilkjent ytelse for behandlingId=${it}")` på slutten inni funksjonen her? Greit å feile hvis vi påstår at vi skal ha forrigeBehandlingId, men den ikke finnes i systemet?

Reply:
Ja, la til denne